### PR TITLE
JW-605 Specify vswitch name as cmd line argument

### PR DESCRIPTION
**Pull request by m-kostrzewa**

### CODE REVIEW

**Comment by m-kostrzewa** [comment]
Janie, przetestuj proszę

File: main.go [hunk 1]
```diff
@@ -19,6 +19,8 @@ func main() {
 		"port of Contrail Controller API")
 	var logLevelString = flag.String("logLevel", "Info",
 		"log verbosity (possible values: Debug|Info|Warn|Error|Fatal|Panic)")
+	var vswitchName = flag.String("vswitchName", "Layered Ethernet0",
```
Comment (sodar) [inline main.go]: In May 2017 update of Windows Server vswitch names created alongside HNS networks are named `Layered_<ifname>`. Maybe we could build `vswitchName` using provided interface name?

File: driver/driver_test.go [hunk 2]
```diff
@@ -31,13 +31,16 @@ import (
...
 var netAdapter string
+var vswitchName string
 var controllerAddr string
 var controllerPort int
 var useActualController bool
 
 func init() {
 	flag.StringVar(&netAdapter, "netAdapter", "Ethernet0",
 		"Network adapter to connect HNS switch to")
+	flag.String("vswitchName", "Layered Ethernet0",
```
Comment (sodar) [inline driver/driver_test.go]: In May 2017 update of Windows Server vswitch names created alongside HNS networks are named `Layered_<ifname>`.

**Review by sodar — CHANGES_REQUESTED** [review]

File: driver/driver_test.go (hunk 2 above)
Comment (m-kostrzewa) [reply driver/driver_test.go]: You mean that the default should have `_`?

**Review by sodar — APPROVED** [review]

**Pull request merged**